server/src/main/java/org/elasticsearch/index/codec/tsdb/ES87TSDBDocValuesProducer.java

```diff
@@ -616,6 +815,32 @@ private static void readNumeric(IndexInput meta, NumericEntry entry) throws IOEx
         }
     }
 
+    private BinaryEntry readBinary(IndexInput meta) throws IOException {
+        final BinaryEntry entry = new BinaryEntry();
+        entry.dataOffset = meta.readLong();
+        entry.dataLength = meta.readLong();
+        entry.docsWithFieldOffset = meta.readLong();
+        entry.docsWithFieldLength = meta.readLong();
+        entry.jumpTableEntryCount = meta.readShort();
+        entry.denseRankPower = meta.readByte();
+        entry.numDocsWithField = meta.readInt();
+        entry.minLength = meta.readInt();
+        entry.maxLength = meta.readInt();
+        if (entry.numDocsWithField > 0) {
+            entry.addressesOffset = meta.readLong();
+
```

Comment:
this empty line looks odd here?

Reply:
removed this as well too: c7e78571c09693fb9cb84618604605b76c4ff2ae